src/chi-vue/src/constants/types.ts

```diff
@@ -48,6 +48,7 @@ export interface DataTableRow {
   print?: {
     isNestedContentPrintDisabled?: boolean;
   };
+  state?: string;
```

Comment:
I would have something more restrictive like:

`export const DATA_TABLE_ROW_STATES = ['success', 'warning', 'danger', 'info'] as const;`
`export type DataTableRowStates = typeof DATA_TABLE_ROW_STATES[number];`

```suggestion
  state?: DataTableRowStates;
```